.github/workflows/pull-compliance.yml

```diff
@@ -1,4 +1,4 @@
-name: "Pull: Compliance Tests"
+name: Compliance
```

Comment:
I had the prefix so that they were all grouped together in the list on the actions page.